bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/channeltypes.xml

```diff
@@ -74,6 +82,10 @@
 		<label>Duration</label>
 		<description>Duration of the program running on the appliance</description>
 		<category>Time</category>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
Should measurement be used in this context? It can be either calculated duration, or remaining time of already running program (also an estimate based on a calculation).

Reply:
Indeed. So what do you suggest? Not that the channel type’s state read only flag may (indeed it **_should_**) inform us if the tag is a `Measurement` or a`Control`. I have a strong feeling that many developers forget to apply the read only flag.

Reply:
Proposing `Calculation.Duration`

Reply:
Calculations seems a bit confusing. When you select a program, a duration will be given for that program. For some appliances and programs, based on sensor input, this duration can be reestimated over time, for example when running a tumble dryer and the humidity level decreases at a certain pace. For other program, it will just be executed without any changes, so the duration will decrease linearly.

Reply:
I think this becomes `Status.Progress` (new property tag).

Reply:
I think the one after is now `Status.Progress` whereas this one should probably be `Status.Duration`  (or ev. `Status.Info`).